src/Simulation/Simulators/ResourcesEstimator/ResourcesEstimator.cs

```diff
@@ -1,201 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using System.Data;
-using System.Diagnostics;
-using System.Linq;
-using System.Reflection;
-using System.Text;
-using Microsoft.Quantum.Simulation.QCTraceSimulatorRuntime;
-using Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators;
-
-namespace Microsoft.Quantum.Simulation.Simulators
-{
-    /// <summary>
-    /// The ResourceEstimator estimates statistics about how many resources
-    /// the given operation needs for execution.
-    /// The resources it calculates are:
-    /// <ol>
-    ///   <li>Primitive operations count</li>
-    ///   <li>Qubits depth (max number of qubits allocated at any given point)</li>
-    ///   <li>Gates width (total number of gates used for the computation)</li>
-    /// </ol>
-    /// </summary>
-    public partial class ResourcesEstimator : QCTraceSimulator
```

Comment:
@DmitryVasilevsky, @msoeken,
Please Make sure that this PR removes all the deprecation messages added in [here](https://github.com/microsoft/qsharp-runtime/pull/1126/files).